iTerm/README.md

```diff
@@ -1,28 +1,28 @@
 # iTerm2
-[iTerm2](http://www.iterm2.com/) is an open source replacement for Apple's Terminal. It's highly customizable and comes with a lot of useful features.
+> [iTerm2](http://www.iterm2.com/) is a replacement for Terminal and the successor to iTerm. It works on Macs with macOS 10.10 or newer. iTerm2 brings the terminal into the modern age with features you never knew you always wanted.
```

Comment:
This is taken straight from http://www.iterm2.com/ not sure about this myself.

Reply:
It sounds very _selly_ and _enterprisy_. I prefer the previous version, saying it's "highly customizable" and has "useful features" says more than "brings the terminal into the modern age with features you never knew you always wanted", in my opinion.

Reply:
Yea I will change that back.